releases/efs-provisioner.yaml

```
@@ -0,0 +1,66 @@
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+releases:
+
+################################################################################
+## EFS provisioner #############################################################
+################################################################################
+
+#
+# References:
+#   - https://github.com/helm/charts/blob/master/stable/efs-provisioner/values.yaml
+#   - https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs
+#
+- name: "efs-provisioner"
+  namespace: "kube-system"
+  labels:
+    chart: "efs-provisioner"
+    repo: "stable"
+    component: "efs"
+    namespace: "kube-system"
+    vendor: "kubernetes"
+    default: "false"
+  chart: "stable/efs-provisioner"
+  version: "0.1.3"
+  wait: true
+  values:
+    - global:
+        deployEnv: '{{ env "DEPLOY_ENV" | default "dev" }}'
+      image:
+        repository: '{{ env "EFS_PROVISIONER_IMAGE_REPOSITORY" | default "quay.io/external_storage/efs-provisioner" }}'
+        tag: '{{ env "EFS_PROVISIONER_IMAGE_TAG" | default "v0.1.2" }}'
+        pullPolicy: "IfNotPresent"
+      resources:
+        limits:
+          cpu: '{{ env "EFS_LIMITS_CPU" | default "200m" }}'
+          memory: '{{ env "EFS_LIMITS_MEMORY" | default "128Mi" }}'
+        requests:
+          cpu: '{{ env "EFS_REQUESTS_CPU" | default "100m" }}'
+          memory: '{{ env "EFS_REQUESTS_MEMORY" | default "128Mi" }}'
+      rbac:
+        ### Optional: RBAC_ENABLED;
+        create: '{{ env "EFS_RBAC_ENABLED" | default "false" }}'
```

This should not be prefixed. RBAC can be enabled globaly
```suggestion
        create: '{{ env "RBAC_ENABLED" | default "false" }}'
```